### PR TITLE
Replace subkey with go-subkey port

### PR DESCRIPTION
**Pull request by MakMuftic**
Replace calling `subkey` command through cmd with [go-subkey port](https://github.com/vedhavyas/go-subkey) so this lib is not dependent on the user to install `subkey`.

I also removed installing `subkey` inside docker build

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,17 +1,11 @@
 # Note: We don't use Alpine and its packaged Rust/Cargo because they're too often out of date,
 # preventing them from being used to build Substrate/Polkadot.
 
-# First Phase - Load Subkey
-FROM parity/subkey:2.0.0 as subkey
-RUN subkey --version
-
 ## Second Phase - Build context for tests
```
Comment (mpetrunic) [inline Dockerfile]:
```suggestion
## First Phase - Build context for tests
```

File: gethrpc/subscription_test.go [hunk 2]
```diff
@@ -19,6 +19,7 @@ package rpc
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/ethereum/go-ethereum/common/hexutil"
```
Comment (mpetrunic) [inline gethrpc/subscription_test.go:22]: -.-
Can you use https://golang.org/pkg/encoding/hex/ or at least some util from polkadot repo

File: client/client.go [hunk 3]
```diff
@@ -52,7 +52,7 @@ func (c client) URL() string {
 }
 
 // TODO move to State struct
```
Comment (mpetrun5) [inline client/client.go:54]: Can we remove this commented block of code?
We can write issue it is valid.

File: rpcmocksrv/server.go [hunk 4]
```diff
@@ -50,8 +50,8 @@ func New() *Server {
 }
 
 func randomPort() int {
-	rand.Seed(time.Now().UnixNano())
```
Comment (mpetrun5) [inline rpcmocksrv/server.go:53]: I don't this know if this needs to be truly random, but we need to keep rand.Seed if it does probably

**Review by mpetrunic — CHANGES_REQUESTED** [review]

File: rpcmocksrv/server.go (hunk 4 above)
Comment (MakMuftic) [reply rpcmocksrv/server.go:53]: I made this change as the linter had a problem with **weak random** used before, but this is part of `rpcmocksrv` so I wouldn't worry about this

**Review by mpetrun5 — APPROVED** [review]

**Pull request merged**